scripts/data_files/version_features.fmt

```diff
@@ -44,6 +44,12 @@ int mbedtls_version_check_feature(const char *feature)
         return -1;
     }
 
+    if (strncmp(feature, "MBEDTLS_", 8)) {
```

Comment:
At some point we should make a PSA equivalent. But it might not be the same C function anyway. And if it is we can have one table per prefix.